hosting/src/hooks/useTanamDocuments.tsx

```diff
@@ -60,6 +60,67 @@ export function useTanamDocuments(documentTypeId?: string): UseTanamDocumentsRes
   return {data, error};
 }
 
+type RecentField = "createdAt" | "updatedAt" | "publishedAt";
```

Comment:
There is no field `createdAt` but  `created` in documents in ruang-tempo, not missing Firestore indexes that causing not displaying data

IMG
<img width="1135" alt="image" src="https://github.com/oddbit/tanam/assets/18025983/e3ef0877-af47-4bd8-9342-f24cfaca69a9">

Result:
<img width="638" alt="image" src="https://github.com/oddbit/tanam/assets/18025983/42602f90-e6f4-40ea-8a58-2ba8098491ff">


TO DO: Fix `TypeError: Cannot read properties of null (reading 'toDate')`

Sorry, I can make this fix, running out of the time 🙂

Reply:
Seems like we need to create migration script to add a new field `createdAt` and get value from legacy data `created` field and still keeping the legacy data to avoid breaking in the app that still uses legacy data

Reply:
No migration script needed. It's waste of time.

Just delete old documents and create new ones. Some models are already implemented for being compliant with reading old documents
 https://github.com/oddbit/tanam/blob/e2a4fa71609f5bba2040aff3a7310158657b09ae/hosting/src/models/TanamDocument.ts#L57-L58